include/language.php

```diff
@@ -40,7 +40,8 @@
  *     custom_flag_url?: string,
  *     custom_flag?: string,
  *     page_on_front:positive-int,
- *     page_for_posts:positive-int
+ *     page_for_posts:positive-int,
+ *     active?: bool
```

Comment:
Should we consider active is optional if it is intialized to true?

Reply:
I would disagree. Since [you always set it in the factory](https://github.com/polylang/polylang/pull/1191/files/c9576921acff74f6cfd78018a87d06029b6e8dec#diff-b79b7331e30aa849724ec6eb266c88b82d91510c53ca648d39262207aa88a9b3R125), I would require it.

Reply:
But [it has a default value](https://github.com/polylang/polylang/pull/1191/files/c9576921acff74f6cfd78018a87d06029b6e8dec#diff-15448452fcf2eda2e4d5c61d73ff632a908cfa35d6fb0ce7b0162654dd0c658eR213), so there is no reason to make it required.

Reply:
🍿

Reply:
Made it mandatory in 143b26a